warehouse/core/src/main/java/datawave/util/TableNames.java

```diff
@@ -0,0 +1,21 @@
+package datawave.util;
+
+/**
+ * Class that contains default table names
+ */
+public class TableNames {
```

Comment:
interface instead of class, and you won't have to declare public static final on the members (since they already will be)

Reply:
Good call, updated in next commit.